src/main/java/uk/gov/laa/ccms/caab/model/ResultsDisplay.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.laa.ccms.caab.model;
+
+import java.util.List;
+import lombok.Data;
+
+/**
+ * Generic class for encapsulating paginated results display.
+ *
+ * @param <T> the type of elements in the results
+ */
+@Data
+public class ResultsDisplay<T> {
```

Comment:
Isn't this class just a copy of the standard Page class?
Can't we just reuse Page?

Reply:
No as we that would require the api version of the object and its impossible to generate classes with <T> the whole point was to remove duplication.